pkg/controller/namespace/namespace_controller_test.go

```diff
@@ -2,9 +2,11 @@ package namespace
 
 import (
 	"context"
-	"github.com/openshift/dedicated-admin-operator/pkg/dedicatedadmin"
+	dedicatedadminproject "github.com/openshift/dedicated-admin-operator/pkg/dedicatedadmin/project"
 	"testing"
 
+	"github.com/openshift/dedicated-admin-operator/config"
```

Comment:
For consistency, it's probably a good idea to call this `operatorconfig`, like in `main.go`.

Reply:
I only renamed it in main because of a conflict, but there's no functional reason not to make this change.  So.. ok.

Reply:
I'm not sure if there's a common go practice for keeping it consistent across packages or not. If there is, I thought to highlight it.